Software README.md update (#31)
ucb-bar/sha3: software/README.md

# Getting Started

> Since SHA3 accelerator uses custom instruction not supported by the vanilla
> toolchain, these workloads require a modified version of Spike simulator to
> build, which can be found [here](https://github.com/ucb-bar/esp-isa-sim). If
> you are using Chipyard, you should install the esp-tools toolchain which comes
> with the appropriate version of Spike.

Reviewer: Just to correct another error in the old README: The workloads can be built with the regular toolchain since the RoCC instructions are encoded with the `.insn` directive.

```suggestion
Running these workloads require a modified Spike simulator that supports the SHA3
custom extension, which can be found [here](https://github.com/ucb-bar/esp-isa-sim). If
you are using Chipyard, you should install the esp-tools toolchain which comes
with the appropriate version of Spike.
```
